operating/examine-images-for-vulnerabilities.adoc

```diff
@@ -82,7 +82,14 @@ include::modules/accessing-delegated-image-scanning.adoc[leveloffset=+1]
 
 include::modules/enhancing-image-scanning-by-accessing-delegated-image-scanning.adoc[leveloffset=+2]
 
-include::modules/configuring-delegated-image-scanning.adoc[leveloffset=+2]
+include::modules/enabling-seamless-image-access-and-efficient-use-of-multiple-registry-sources.adoc[leveloffset=+2]
```

Comment:
Believe this will read better if the mirroring section appears either before or after the sections describing how to setup delegated scanning (currently it splits it in the middle).  For example (assuming other PR comments addressed):

Here is the list of headings on the page:
```
About RHACS Scanner V4
Scanning images
    Understanding an...
    Supported packag...
    Supported progra...
    Supported runtim...
    Supported operat...
    Registry Mirrors      <--- **MIRRORING SECTION MOVED HERE BEFORE the delegated scanning section**
Accessing delegated image scanning
    Enhancing image ...
    Enabling seamles...   <--- CURRENT location may distract from setting up delegated scanning
    Scanning images ...
    Installing and c...
    Enabling seamles...   <--- **MIRRORING SECTION MOVED HERE AFTER delegated scanning setup described**
Setting up scanning
    Automatic scanni...
    Scanning inactiv...
About vulnerabilities
    Fetching vulnera...
    Understanding vu...
Disabling language-specific vulnerability scanning
Additional resources
```